quantum/keymap_extras/keymap_spanish_latam.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 QMK
```

Comment:
Implementing directly is invalid. Content is now configured within `data/constants/keycodes/extras/` and generated

Reply:
Done